### ISSUE TITLE
Transformer Encoder become very slaggy

### ISSUE THREAD

**Issue by nan-wang** [jina-ai/jina#774]
**Describe the bug**
Transformer Encoder become very slaggy for encoding.

**Describe how you solve it**
#764 The lazy loading is removed and for each call of `encode` the model and the tokenizer is reloaded.

https://github.com/jina-ai/jina/pull/764/files#diff-a59839a922c1305451aac603a06bc1d0

- - -

**Environment**

**Screenshots**

┆Issue is synchronized with this [Jira Task](https://jinaai.atlassian.net/browse/JINACORE-150) by [Unito](https://www.unito.io/learn-more)

**Comment by JoanFM** [comment]
Seems like `@cached_property` is not working here?

**Comment by nan-wang** [comment]
Yes, I'll test it.

**Comment by rohan1chaudhari** [comment]
I think the function are over-ridden in the child classes by a non- "cached_property" function.
For example : https://github.com/jina-ai/jina/pull/764/files#diff-a59839a922c1305451aac603a06bc1d0 (line 175)

**Comment by rohan1chaudhari** [comment]
```
class cached_property(object):
    def __init__(self, func):
        self.func = func

    def __get__(self, obj, cls):
        cached_value = obj.__dict__.get(f'CACHED_{self.func.__name__}')
        if cached_value is not None:
            return cached_value

        value = obj.__dict__[f'CACHED_{self.func.__name__}'] = self.func(obj)
        return value
```

**Comment by JoanFM** [comment]
Has it been solved by #785 ?

**Comment by nan-wang** [comment]
yes, resolved by #785
